templates/admin/streams/edit.html.twig

```diff
@@ -0,0 +1,44 @@
+{% extends 'admin/admin.html.twig' %}
+
+{% block content %}
+
+	<div class="row">
+		<div class="col s12">
+			<h2>{{ stream.streamName }}
```

Comment:
we should close the H2 on this line. 
If you want to put everything in the same container, maybe we should use a div between the <h2>, <h3> and the image.